tools/linter_lib/custom_check.py

```diff
@@ -131,7 +131,9 @@ def build_custom_checkers(by_lang):
          'description': 'Fix trailing whitespace'},
         {'pattern': '^#+[A-Za-z0-9]',
          'strip': '\n',
-         'description': 'Missing space after # in heading'},
+         'description': 'Missing space after # in heading',
+         'tests_good': ['### some heading\n', '# another heading\n'],
+         'tests_bad': ['###some heading\n', '#another heading\n']},
```

Comment:
We probably should not require the trailing newlines here, right?  Seems like it'll just clutter up the tests.

Reply:
Yeah, I guess. I suspected there to be some rules testing for just those newlines at the end, but we would probably wanta special test case for those.

Reply:
Yeah, those are special in the code anyway, and can be special in tests.  Best to make the test framework make life good for the common cases.